File: app/src/main/java/se/appshack/android/refactoring/main/data/model/PokemonSpritesModel.java
package se.appshack.android.refactoring.main.data.model;

import com.google.gson.annotations.SerializedName;

public class PokemonSpritesModel {

    @SerializedName("front_default")
    public String urlFront;

    @SerializedName("back_default")
    public String urlBack;
}

File: app/src/main/java/se/appshack/android/refactoring/main/data/model/PokemonSpritesModel.kt
package se.appshack.android.refactoring.main.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class PokemonSpritesModel (

    @SerializedName("front_default")
    var urlFront: String? = null,

    @SerializedName("back_default")
    var urlBack: String? = null

): Parcelable

